### PR TITLE
CAMEL-13877 fix NPE in RouteHealthCheck

### PR DESCRIPTION
**Pull request by pheaber**
Fix for NPE

### CODE REVIEW

File: core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java [hunk 1]
```diff
@@ -84,21 +84,23 @@ protected void doCall(HealthCheckResultBuilder builder, Map<String, Object> opti
                 builder.up();
             }
 
-            if (builder.state() != State.DOWN) {
-                // If JMX is enabled, use the Managed MBeans to determine route
-                // health based on performance counters.
-                ManagedRouteMBean managedRoute = context.getExtension(ManagedCamelContext.class).getManagedRoute(route.getId());
+			if (builder.state() != State.DOWN) {
```
Comment (oscerd) [inline core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java]: I think the indentation is wrong. Please run this with the profile sourcecheck enabled. You should see warnings.

Comment (pheaber) [reply core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java]: OK, I'll update it

Comment (pheaber) [reply core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java]: I've run this from camel/core/camel-base but it's not reporting any errors. Is there something else I should check?
`mvn clean install -Psourcecheck`

**Comment by oscerd** [comment]
I'll double check tomorrow. Btw you should see warnings and not errors

File: core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java (hunk 1 above)
Comment (pheaber) [reply core/camel-base/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java]: Oh, I wasn't looking for warnings. I'll fix them now. Sorry for my confusion.

**Comment by oscerd** [comment]
Thanks, it has been merged

**Pull request closed without merging**